static_src/stores/service_instance_store.js

```diff
@@ -37,6 +37,15 @@ class ServiceInstanceStore extends BaseStore {
         break;
       }
 
+      case serviceActionTypes.SERVICE_INSTANCE_RECEIVED: {
+        const instance = this.formatSplitResponse(
+          [action.serviceInstance])[0];
```

Comment:
@jcscottiii I thought `formatSplitResponse` only works with arrays but you might be right.